packages/node_modules/@webex/plugin-devicemanager/src/devicemanager.js

```diff
@@ -0,0 +1,503 @@
+/*!
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '@ciscospark/internal-plugin-lyra';
+import '@ciscospark/internal-plugin-search';
+import {SparkPlugin} from '@ciscospark/spark-core';
+import {LYRA_SPACE, UC_CLOUD, DEFAULT_PRODUCT_NAME} from './constants';
+import DeviceCollection from './collection';
+import uuid from 'uuid';
+
+const DeviceManager = SparkPlugin.extend({
+  namespace: 'DeviceManager',
+  _devicePendingPinChallenge: null,
+  _pairedDevice: null,
+  _boundSpace: null,
+
+  initialize() {
+    // Lyra mercury events listener
+    this.spark.internal.mercury.on('event:lyra.space_updated', ({data}) => {
+      this._receiveDeviceUpdates(data);
+    });
+  },
+
+  /**
+  * Gets a list of all recent devices associated with the user
+  * the device list gets populated from Redis
+  * @returns {Promise<Device>}
+  */
+  list() {
+    return DeviceCollection.list();
+  },
+
+  /**
+  * Gets a list of all recent devices associated with the user
+  * the device list gets populated from Redis
+  * @returns {Promise<Device>}
+  */
+  refresh() {
+    DeviceCollection.reset();
+    return this.spark.request({
+      api: 'wdm',
+      resource: 'devices/auxiliary'
+    })
+      .then((res) => {
+        if (!res.body) {
+          return Promise.reject();
+        }
+        return Promise.all(res.body.items.map((device) => {
+          if (device.deviceClass === UC_CLOUD) {
+            device.id = `${this.spark.internal.device.services.wdmServiceUrl}/${device.id}`;
+          }
+          return this._decryptDeviceName(device);
+        }))
+          .then(() => {
+            /* eslint-disable consistent-return */
+            res.body.items.forEach((device) => {
+              if (device.deviceInfo
+                && device.deviceInfo.machineType === LYRA_SPACE) {
+                return this.spark.internal.lyra.space.get(device.deviceInfo)
+                  .catch((err) => {
+                    this.logger.error('DeviceManager#refresh: failed to receive device info', err);
+                  });
+              }
+            });
+            /* eslint-enable consistent-return */
+            res.body.items.forEach((device) => {
+              DeviceCollection.set(device);
+            });
+            return this.list();
+          });
+      })
+      .catch((err) => {
+        this.logger.error('DeviceManager#refresh: failed to fetch recent devices', err);
+      });
+  },
+
+  /**
+  * Search for a device by name
+  * @param {Object} options
+  * @param {string} options.searchQuery
+  * @returns {Promise<Device>}
+  */
+  search(options) {
+    if (!options || !options.searchQuery) {
+      this.logger.error('DeviceManager#search: options.searchQuery is required');
+      return Promise.reject(new Error('DeviceManager#search: options.searchQuery is required'));
+    }
+    return this.spark.internal.search.people({
```

Comment:
yes, since it is an argonaut search, i think it was assumed as people search initially and hence the function was named as people(). Will leave it as is for now as it might be used in several other apps.